executor/admin.go

```diff
@@ -449,6 +471,13 @@ func (e *RecoverIndexExec) Next(ctx context.Context, req *chunk.Chunk) error {
 		return nil
 	}
 
+	recoveringClusteredIndex := e.index.Meta().Primary && e.table.Meta().IsCommonHandle
+	if recoveringClusteredIndex {
+		req.AppendInt64(0, 0)
+		req.AppendInt64(1, 0)
+		e.done = true
+		return nil
+	}
```

Comment:
Should we move this below check the partition table?

Reply:
Even if it is a partition table, recovering a clustered primary key should not have an effect.

Reply:
OK.